values/velero/velero.gotmpl

```diff
@@ -0,0 +1,69 @@
+{{- $v := .Values }}
+{{- $V := $v.apps.velero }}
+{{- $i := ### }}
+
+### Image tags dynamically added once provider is identified (ie. aws:v1.5.0, csi:v0.3.0)
+
+
+global:
+
+velero:
+  logLevel: {{ $V.logLevel }}
+
+image:
+  repository: velero/velero
+  tag: {{ $V.image.tag }}
+  pullPolicy: {{ $V.image.pullPolicy }}
+  imagePullSecrets
+
+resources:
+  requests:
+    cpu: 500m
+    memory: 128Mi
+  limits:
+    cpu: 1000m
+    memory: 512Mi
+
+# @TODO Depends on service provider
+initContainers:
+  # IF 
+  - name: velero-plugin-for-aws
+    image: velero/velero-plugin-for-aws:v1.5.0
+    imagePullPolicy: IfNotPresent
+    volumeMounts:
+      - mountPath: /target
+        name: plugins
+
+# Set fsGroup for `AWS IAM Roles for Service Accounts`. Only AWS.
+podSecurityContext: {}
+  # fsGroup: 1337
+
+metrics:
+  enabled: true
+  scrapeInterval: 30s
+  scrapeTimeout: 10s
+
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: velero
+spec:
+  groups:
+  - name: velero-failures
+    rules:
+    - alert: VeleroBackupPartialFailures
+      annotations:
+        message: Velero backup {{ $labels.schedule }} has {{ $value | humanizePercentage }} partialy failed backups.
+      expr: |-
+        velero_backup_partial_failure_total{schedule!=""} / velero_backup_attempt_total{schedule!=""} > 0.25
+      for: 15m
+      labels:
+        severity: warning
+    - alert: VeleroBackupFailures
+      annotations:
+        message: Velero backup {{ $labels.schedule }} has {{ $value | humanizePercentage }} failed backups.
+      expr: |-
+        velero_backup_failure_total{schedule!=""} / velero_backup_attempt_total{schedule!=""} > 0.25
+      for: 15m
+      labels:
+        severity: warning
```

Comment:
You can't just paste whole manifests into values files. Please add PrometheusRules to `values/prometheus-operator/rules/*` and optionally include them in [values/prometheus-operator.gotmpl:L119](https://github.com/redkubes/otomi-core/blob/main/values/prometheus-operator/prometheus-operator.gotmpl#L119)

Reply:
Got it 👍